src/com/levi/BattleshipApplication.java

```diff
@@ -1,23 +1,40 @@
 package com.levi;
 
+import com.levi.model.Board;
+import com.levi.model.Player;
+
 /**
  * Main, starts the game.
- * @version 3.0
+ * @version 4.0
  * @since 2019
  * @author Levi Rosal
  */
 public class BattleshipApplication {
+
     /**
-     * Gets names and locations of files and starts the game.
+     * Gets names and locations of files, starts and finish the game.
      */
     public static void main(String[] args) {
-        BattleshipLaunch game = new BattleshipLaunch();
-        String nameP1 = args[0].substring(10,(args[0].length()));
-        String locP1 = args[1].substring(10,(args[1].length()));
-        String nameP2 = args[2].substring(10,(args[2].length()));
-        String locP2 = args[3].substring(10,(args[3].length()));
+        Player playerOne = new Player();
+        Player playerTwo = new Player();
+
+        for (int i = 0; i < args.length ; i++) {
```

Comment:
Extract to a method and explain what this `for` do.

Reply:
Done, commit 0d524b0.